cmd/share-createlink.go

```diff
@@ -0,0 +1,147 @@
+// Copyright © 2017 Dropbox, Inc.
+// Author: Daniel Porteous
```

Comment:
For consistency, can you omit this line? `git blame` can always reveal authors :)